doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,56 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which support separation of concerns during image build. A fully integrated security testing should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issue.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
+
+The following functionalities are recommended for securing kubernetes platform;
+- Image Signing
+- Role-base Access Control
+- Secret Managment
+- Platform multitenant security
```

Comment:
Is it / could it be described further what kind of multi-tenancy is meant here e.g. devops teams in single organization sharing a cluster?  Vanilla Kubernetes is still lacking support for real multi-tenancy as discussed in detail e.g. in [this post](https://blog.jessfraz.com/post/hard-multi-tenancy-in-kubernetes/).

Reply:
Resolved